### ISSUE TITLE
Popular search terms only updated after clicking the search icon

### ISSUE THREAD

**Issue by BendixSpring** [TeamMentor/TM_4_0_Design#338]
With AJAX and all, I was expecting the popular search terms to be dynamically updated as I am sitting on a page.  This way we get to see all kinds of fun things:
![image](https://cloud.githubusercontent.com/assets/3578332/5945694/6e1b507c-a6fe-11e4-9c2c-0518180a512c.png)

**Comment by DinisCruz** [comment]
Well AJAX will be a bit harder to do on this version, since there is no Javascript in there :) 

@roman87 @luchia have we captured on the Flare ideas/plans the 'dynamic updating of popular search terms'?

**Comment by roman87** [comment]
@DinisCruz yep we do. This is tracked in the story related above. Currently set for Sprint 4
